### PR TITLE
Add `isort.check` setting

### PR DESCRIPTION
**Pull request by karthiknadig**
closes https://github.com/microsoft/vscode-python/issues/20029

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "settings.logLevel.off.description": "Most logging is turned off, any information that is always logged might still be shown.",
     "settings.logLevel.warn.description": "Includes all messages in the error category and any additional warnings.",
     "settings.args.description": "Arguments passed in. Each argument is a separate string in the array.",
+    "settings.check.description": "Runs `isort` in check mode, and reports import sort issues.",
```
Comment (luabud) [inline package.nls.json]: just to make sure I understand it correctly: `check` will control whether the "imports are not correctly sorted" warning, is that correct? 
Maybe something like "Controls whether to run `isort` and report import sort issues"?

Comment (karthiknadig) [reply package.nls.json]: That is correct, this is for showing the error/warning that imports are not sorted. I will update the text.

Comment (karthiknadig) [inline package.nls.json]:
```suggestion
    "settings.check.description": "Controls whether to run `isort` and report import sort issues.",
```

**Review by karrtikr — APPROVED** [review]
LGTM. Removed periods from PR title as it appears weirdly in changelogs.

image

**Pull request merged**